app/frontend/containers/CallToolView/CallToolView.js

```diff
@@ -116,8 +121,14 @@ class CallToolView extends Component {
     });
   }
 
-  selectNewTarget(targetCountryCode: string) {
-    const candidates = _.filter(this.props.targets, t => { return t.countryCode === targetCountryCode; });
+  selectNewTarget(targetCountryCode: ?string = null) {
+    let candidates;
```

Comment:
_Personally_ I'd go for `let` over `var` since block scope is easier to reason about than function scope.